src/tests/tasks.test.js

```diff
@@ -1,18 +1,96 @@
 import request from 'supertest';
-import Project from '../models/Projects';
-import projectsSeeds from '../seeds/projects';
 import app from '../app';
+import Tasks from '../models/Tasks';
+import task from '../seeds/tasks';
 
 beforeAll(async () => {
-  await Project.collection.insertMany(projectsSeeds);
+  await Tasks.collection.insertMany(task);
 });
 
-describe('Test Projects routes', () => {
-  test('It should get the project list', async () => {
-    const response = await request(app).get('/projects');
-    expect(response.body.message).toBe('Project found');
+describe('Test Tasks routes', () => {
+  test('It should get the tasks list', async () => {
+    const response = await request(app).get('/tasks').send();
+    expect(response.body.message).toBe('Tasks');
     expect(response.statusCode).toBe(200);
     expect(response.body.data.length).toBeGreaterThan(0);
     expect(response.body.error).toBe(false);
   });
+
+  test('It should create task', async () => {
+    const response = await request(app).post('/tasks').send({
+      description: 'Landing page',
+      workedHours: 8,
+      projectId: '6289ae2826280830345a9864',
+      date: '2020-04-09',
+    });
+    expect(response.body.message).toBe('Task created');
+    expect(response.statusCode).toBe(201);
+    expect(response.body.error).toBe(false);
+  });
+
+  test('It should return a status 400', async () => {
+    const response = await request(app).post('/tasks').send();
+    expect(response.body.message).toBe('There was an error');
+    expect(response.statusCode).toBe(400);
+  });
+
+  test('It should return an error because description is too short', async () => {
+    const response = await request(app).post('/tasks').send({
+      description: 'L',
+      workedHours: 8,
+      projectId: '628c3ac49d840d661742bad9',
+      date: '2020-04-09',
+    });
+    expect(response.statusCode).toBe(400);
+  });
+
+  test('It should return a status 404 because id is incorrect', async () => {
+    const response = await request(app).get('/tasks/628a59c9f67d451615818847').send();
+    expect(response.body.message).toBe('Task not found');
+    expect(response.statusCode).toBe(404);
+    expect(response.body.error).toBe(true);
+  });
+
+  test('It should return a task by id', async () => {
+    const response = await request(app).get('/tasks/6289adc53ba4baea7bf2defd').send();
+    expect(response.body.message).toBe('Tasks');
+    expect(response.statusCode).toBe(200);
+    expect(response.body.error).toBe(false);
+  });
+
+  test('It should return a status 404 because id is incorrect', async () => {
+    const response = await request(app).put('/tasks/628ae081dbe588f5677e9982').send({
+      description: 'Landing Page',
+      workedHours: 8,
+      projectId: '628c3ac49d840d661742bad9',
+      date: '2020-04-09',
+    });
+    expect(response.body.message).toBe('Task not found');
+    expect(response.statusCode).toBe(404);
+  });
```

Comment:
Y todo esto iría dentro de otro describe que sea
```suggestion
  describe('GET /:id', () =>{
  test('It should return a task by id', async () => {
  });
  });
```

Reply:
Está re bien igual, si dividis eso de los describe para que sea mas facil de leer ya estaría joya 🚀

Reply:
+2

Reply:
+3